google/ggrc-core
# [5Obj] GGRC-8496 Disable editing 5 objects

# Comment

**Should be merged to "GGRC-7200".**

# Issue description

Disable editing 5 objects (Contract, Objective, Policy, Requirement, Threat)

# Steps to test the changes

1. Open InfoPane for each object and check Three Dots Menu. User should not be able to Delete and Edit object.
2. Open Dashboard, click on 'Create object'. Contract, Objective, Policy, Requirement and Threat should be disabled.
3. Check LHN, Global Search. 'Create' 5 objects should be disabled.


# Solution description

Add 'ChangeableExternally' mixin to 5 models (Objectives, Requirements, Contracts, Policies, Thread).
Update and rename isDirective getter in tree-item-extra-info.js.

# Sanity checklist




# PR Review checklist
